modules/uhttp/client/client_filters.go

```diff
@@ -25,32 +25,48 @@ import (
 
 	"go.uber.org/fx"
 	"go.uber.org/fx/auth"
+	"go.uber.org/fx/config"
 	"go.uber.org/fx/internal/fxcontext"
 
 	"github.com/opentracing/opentracing-go"
 	"github.com/opentracing/opentracing-go/ext"
 )
 
-// Filter applies filters on client requests, request contexts such as
-// adding tracing to the context
+// Sender sends the http request. It is a simple wrapper around http.RoundTripper, so
+// Send must be safe to use by multiple go routines
+type Sender interface {
+	Send(ctx fx.Context, r *http.Request) (resp *http.Response, err error)
```

Comment:
I prefer Do instead of Send, it matches the http naming. Less mental overload on engineers. What do you think?